pkg/config/generate.go

```diff
@@ -410,16 +411,6 @@ func Generate(cs *acsapi.OpenShiftManagedCluster) (err error) {
 			kubeconfig: &c.NodeBootstrapKubeconfig,
 			namespace:  "openshift-infra",
 		},
-		{
-			clientKey:  c.Certificates.Admin.Key,
-			clientCert: c.Certificates.Admin.Cert,
-			// sync kubeconfig has the same capabilities as admin kubeconfig, only difference
-			// is the use of HCP internal DNS to avoid waiting for the Azure loadbalancer to
-			// come up in order to start creating cluster objects.
-			endpoint:   "master-000000",
```

Comment:
YAY! https://github.com/openshift/openshift-azure/issues/175 closed but re-introduced in previous comment in different form